.mailmap

```diff
@@ -1 +1,2 @@
 Alice Hartley <alicealicehartley@gmail.com> <alicehartleyhartley@gmail.com>
+Christopher Miller <admin@mctherealm.net> <cmiller548@gmail.com> <christophemiller@ufl.edu>
```

Comment:
Nit: No Newline

Reply:
Resolved in a8fafd6fcd0db0777478b95d1dc0f32237f36fef